### PR TITLE
feat(build): add `ready` phase for hardened-plan ready-for-execution state

### PR DESCRIPTION
**Pull request by howarewoo**
## Goal

Give the build loop a status for "plan hardened, ready for execution." After the step-6 plan harden, `woostack-build` now advances the spec to a new `ready` phase instead of leaving it at `planning`.

## Summary

The phase enum jumped `planning → executing` with no ready state, so `planning` ("plan written, 0 boxes done") covered both *not-yet-hardened* and *hardened-and-ready*, and `/woostack-status` kept telling the user to "harden plan" after it was hardened.

Inserts `ready` between `planning` and `executing`:

```
draft → hardened → approved → planning → ready → executing → in-review → done   (+ abandoned)
```

- `planning` = plan written, not yet hardened (step 4, woostack-plan)
- `ready` = plan hardened, 0 boxes done, ready for execution (step 6, woostack-build) ← new

`ready` is a head state: displayed via `resolve_phase`'s authored fall-through, needs no truth-table row, and requires a plan (stays out of the no-plan exemption).

**Files:** `conventions.md` (enum + table row), `status.sh` (VALID_PHASES, `next_action`, status-lags flag group), `test-status.sh`, `spec-template.md`, `woostack-commit/SKILL.md`, `woostack-build/SKILL.md` (authoring at step 6 + constraint).

## Test plan

**Automated**
- `bash skills/woostack-status/scripts/tests/test-status.sh` → 62 passed, 0 failed (was 59; +3 for the `ready` case, added Red-first).
- `bash -n skills/woostack-status/scripts/status.sh` → clean.

**Manual**
- `grep` sweeps confirm every enum source lists `ready` and no stale `planning → executing` / old planning next-action text remains.
- Verified verify-only sites (`woostack-tdd` "≥ planning", `woostack-plan`, README/`action.yml`) need no edit.

Spec: .woostack/fixes/2026-06-12-build-loop-ready-phase.md

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

**Comment by howarewoo** [comment]
* **#310** <a href="https://app.graphite.com/github/pr/howarewoo/woostack/310?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.com/github/pr/howarewoo/woostack/310?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `main`


<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: skills/woostack-status/scripts/tests/test-status.sh [hunk 1]
```diff
@@ -78,6 +78,15 @@ mkspec "$p" echo planning feature/echo
 run_status "$p"
 assert_contains "$OUT" "echo" "echo row present"
 assert_contains "$OUT" "no plan" "0-plan flagged"
+
+# `ready` = plan hardened, 0 boxes done, ready for execution (build step 6).
```
Comment (howarewoo) [inline skills/woostack-status/scripts/tests/test-status.sh:82]: **Nit: Ready phase lacks edge-case coverage**

The new `ready` fixture only covers the happy path: a spec with a plan, no PR, and the next-action text. The diff also adds `ready` to the status-lags guard and relies on `ready` still requiring a plan, but neither boundary path is pinned by the test update.

Fix: Add a planless `ready` fixture that asserts the no-plan warning, and a `ready` fixture with a matching open PR that asserts `status lags`, so the new phase is covered on both boundary paths.

<sub>— <strong>MEDIUM · NIT</strong> · flagged by the <code>tests</code> agent</sub>

**Review by howarewoo — COMMENTED** [review]
## AI Deep Code Review Summary

Reviewed the new `ready` build-loop phase across the status board, build/commit skill docs, and status tests. The implementation is consistent; one non-blocking test boundary gap is worth pinning.

---
**Status: APPROVED** — No blocking findings, 1 nit(s). See inline comments.
*Audited by woostack-review · Host: codex · Provider: openai · Model: gpt-5*



_Review event downgraded to COMMENT — GitHub blocks APPROVE on your own PR. Status line above carries the actual verdict._

File: skills/woostack-status/scripts/tests/test-status.sh (hunk 1 above)
Comment (howarewoo) [reply skills/woostack-status/scripts/tests/test-status.sh:82]: Fixed in 2b165e18836350f6c999816eed70baee72718de3.
